Microsoft.Toolkit.Uwp.UI.Controls/MosaicControl/MosaicControl.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style TargetType="controls:MosaicControl">
+        <Setter Property="Template">
```

Comment:
For the benefit of Xbox One controller navigation (and keyboard navigation), can we add `<Setter Property="IsTabStop" Value="False" />` to this? Whilst it's content is fine to be focusable I don't think there's any reason the container should be by default.